src/city/components/AddForm.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const AddForm = () => {
+    return(<>City form</>)
```

Comment:
It's better to return <span>City Form</span> or to cover it to another some text tag but not to use it like fragments

Reply:
It's better to return City Form or to cover it to another some text tag but not to use it like fragments